oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -239,6 +239,9 @@ storage:
     group: ${SW_STORAGE_BANYANDB_GROUP:default}
     maxBulkSize: ${SW_STORAGE_BANYANDB_MAX_BULK_SIZE:5000}
     flushInterval: ${SW_STORAGE_BANYANDB_FLUSH_INTERVAL:15}
+    metricsShardsFactor: ${SW_STORAGE_BANYANDB_METRICS_SHARDS_FACTOR:2}
+    recordShardsFactor: ${SW_STORAGE_BANYANDB_RECORD_SHARDS_FACTOR:2}
+    superDatasetShardsFactor: ${SW_STORAGE_BANYANDB_SUPERDATASET_SHARDS_FACTOR:2}
```

Comment:
metricsShardsFactor and recordShardsFactor should be 1 ideally. Because we have one measure/stream per metric/record, one shard should be enough.

Reply:
Fixed